bundle/bin/rke2-uninstall.ps1

```diff
@@ -301,20 +301,43 @@ function Remove-Containerd () {
             }
         }
 
-        # Some resources in the namespace take a while to disappear. Try several times to remove the namespace and give up after 30s 
-        $endTime = (Get-Date).AddSeconds(30)
-        while ((Get-Date) -lt $endTime) {
+        # Resources in the namespace take a while to disappear. Snapshots are always the last to go.
+        # For 240s, snapshots are checked and namespace won't be removed until snapshots are gone.
+        # If timeout is reached, we will try removing snapshots directly. If that does not work, we stop trying and user is warned
+        $endTime = (Get-Date).AddSeconds(240)
+        $forceRemove = $false
+        while ($true) {
             $namespaces = $(Find-Namespaces)
             if ($namespaces) {
                 foreach ($ns in $namespaces) {
-                    Remove-Namespace $ns
+                    $snapshots = $(Find-Snapshots $ns)
+                    if ($snapshots) {
+                        Write-Output "Tasks, containers, images and snapshots still being deleted. Retrying (timeout 240s)"
```

Comment:
We can just say `Retrying resource deletion, waiting for 10s...` and remove timeout from here